extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -849,7 +849,11 @@ private static Pair<TableDefinition, ParquetInstructions> infer(
             allColumns.add(ColumnDefinition.fromGenericType(partitionKey, dataType, null,
                     ColumnDefinition.ColumnType.Partitioning));
         }
-        allColumns.addAll(schemaInfo.getFirst());
+        // Only read non-partitioning columns from the parquet files
```

Comment:
This was needed because the parquet files written by Iceberg had partitioning columns data included in the parquet files. So was leading to duplicate columns.

Reply:
I want to make sure we aren't breaking any existing behavior w/ this change...

Reply:
The tests are passing and I am able to read all the files in deephaven-examples. 
I will wait for Ryan to have a final comment.